src/admin-views/create-organizer-fields.php

```diff
@@ -16,6 +16,10 @@
 	</td>
 	<td class="organizer-email">
 		<input id="organizer-email" type='text' name='organizer[Email][]' class='organizer-email' size='25' value='' />
-		<p><?php esc_html_e( 'The e-mail address will be obfuscated on your site to avoid it getting harvested by spammers.', 'the-events-calendar' ); ?></p>
+		<?php if ( function_exists( 'tribe_is_community_edit_event_page' ) && tribe_is_community_edit_event_page() ) : ?>
```

Comment:
While this works, imho TEC shouldn't know or care about the existence of Community Events functions (we shouldn't code and fix problems for our dependencies, it's a sign things are flowing in the wrong direction). 

I'd much rather the string be filterable (CE can then take advantage of the filter) or, simpler yet, how about simply using the first form of the string which works well in both situations?